python/admin/api.py

```diff
@@ -0,0 +1,62 @@
+import os,subprocess
+from flask import Flask,request,abort,jsonify
+
+valid_properties = ["ssid", "channel", "hostname", "staticsite", "password", "system", "ui-config"]

```

Comment:
Have you verified that setting the ui-config property is working? I had some issues in php with getting the json escaped properly in order to get the file updated properly.

Reply:
I don't think I have. Thanks for pointing this out.

Reply:
thanks for your replies - once you verify this the ui-config prop is working, I think this looks good